### ISSUE TITLE
Could have when the user RSVPs for an event and tries again they get a message.

### ISSUE THREAD

**Issue by Alex112292** [mikepmerritt/Challenge-1#17]
The message could say something along the lines of "you have already RSVP'd for this event would you like to cancel?"

**Comment by howardbyrd** [comment]
Yes it makes sense, right now you get no message if an event overlaps with another. It's just added at the same time as a new line.
